### DISCUSSION TITLE
CircularProgress round track

### DISCUSSION THREAD

**Discussion by bazuka5801** [quasarframework/quasar#14425] [Ideas / Proposals] [answered]
Add  [stroke-linecap](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linecap) to Circular Progress
stroke-linecap="round"

https://github.com/quasarframework/quasar/blob/baf2f9967ae2db0ed1ec483f0ee06263ffd84fcc/ui/src/components/circular-progress/QCircularProgress.js#L69-L82

Example: 
![Screenshot 2022-09-18 at 18 55 43](https://user-images.githubusercontent.com/11452353/190916234-0d795d7c-b6ae-4485-9fc5-6c3e741ec8f8.png)

**Comment by rstoenescu** [comment, marked as answer]
New Boolean prop for QCircularProgress (`rounded`) will be available in Quasar v2.8.4
